solaris/solaris11/SPECS/template_agent_v5.0.0.json

```diff
@@ -807,6 +807,198 @@
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Jul/wazuh-[0-9][0-9].log.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Aug/wazuh-[0-9][0-9].json.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Aug/wazuh-[0-9][0-9].log.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Sep/wazuh-[0-9][0-9].json.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Sep/wazuh-[0-9][0-9].log.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Oct/wazuh-[0-9][0-9].json.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Oct/wazuh-[0-9][0-9].log.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Nov/wazuh-[0-9][0-9].json.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Nov/wazuh-[0-9][0-9].log.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Dec/wazuh-[0-9][0-9].json.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
+    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Dec/wazuh-[0-9][0-9].log.gz": {
+        "class": "dynamic",
+        "group": "wazuh",
+        "mode": "0640",
+        "prot": "-rw-r-----",
+        "type": "file",
+        "user": "wazuh"
+    },
     "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Jan/ossec-[0-9][0-9].json.gz": {
```

Comment:
If we added 
`    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Dec/wazuh-[0-9][0-9].log.gz": {`
we sould remove 
`    "/var/ossec/logs/wazuh/[0-9][0-9][0-9][0-9]/Jan/ossec-[0-9][0-9].json.gz": {`
isn't it?

Reply:
Done in 84747b6